.changelog/3213.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
```

Comment:
This should be `improvement` instead of feature. In Waypoint, we reserve the `feature` release note for brand new components rather than something that is improving on an existing thing.

Reply:
fixed!